File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get input from user
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while(n < 1 || n > 8);

    for(int row = 0; row < n; row++)
    {
        //this loop runs equal to actual row number
        for(int i = 0; i <= row; i++)
        {
            printf("#");
        }
        
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("height: ")
    if height >= 1 and height <= 8:
        break

for i in range(1,height+1):

    for a in range(height - i):
        print(" ",end="")

    for b in range(height - a):
        print("%",end="")

    print("",end ='\n')